pkg/auth/session.go

```diff
@@ -8,7 +8,7 @@ import (
 	"k8s.io/klog"
 )
 
-const openshiftSessionCookieName = "openshift-session-token"
+const OCPAccessTokenCookieName = "openshift-session-token"
```

Comment:
It's not always OCP, so I'd stick with the generic "openshift."

```suggestion
const openshiftAccessTokenCookieName = "openshift-session-token"
```